dnsaas/urls.py

```diff
@@ -8,11 +7,19 @@
 from ui.views import start_page
 
 
+def patterns(prefix, *args):
+    if VERSION < (1, 9):
+        from django.conf.urls import patterns as django_patterns
```

Comment:
actually `patterns` are deprecated since Django 1.8, so do we have to use them at all here? Couldn't we use simple list of urls?